### ISSUE TITLE
MultiMAP graph

### ISSUE THREAD

**Issue by dawe** [Teichlab/MultiMAP#1]
Hi, I have tried MultiMAP to integrate scRNA and scATAC data. I see the object in return is a `AnnData` instance with some slots filled. In particular, in addition to the embeddings (`adata.obsm['X_multimap']`) I believe the graph used by the model is stored in `adata.obsp['connectivities']`, am I correct? Is there a reason why only the unweighted graph is returned? Also, it may be useful to add a slot in `adata.uns['neighbors']` pointing at such connectivity so that one can use scanpy's functions (e.g. to call cell groups)

**Comment by ktpolanski** [comment]
Thanks for pointing this out - during the initial development of BBKNN, the neighbour storage was relatively minimal, so I focused on other stuff here first when I became involved. I think I created an anndata 0.7.0+ compliant form of `.uns` parameter storage now, give it a spin?

The distances graph is a tricky one because of how the method operates. I've brought this to @mikasarkinjain's attention before, but this is apparently nontrivial to make. Tagging him in for a more appropriate discussion partner on the matter.
